Independently created by me, but exactly the same as
https://bugs.freedesktop.org/show_bug.cgi?id=5175#c4

Donnie Berkholz <spyderous@gentoo.org>

? configurable-mapdir.patch
Index: Makefile.am
===================================================================
RCS file: /cvs/xorg/font/util/Makefile.am,v
retrieving revision 1.9
diff -u -b -B -r1.9 Makefile.am
--- Makefile.am	6 Dec 2005 22:48:41 -0000	1.9
+++ Makefile.am	8 Dec 2005 19:48:34 -0000
@@ -31,7 +31,7 @@
 aclocaldir = $(datadir)/aclocal
 aclocal_DATA = fontutil.m4
 
-mapfilesdir = @mapdir@
+mapfilesdir = @MAPDIR@
 mapfiles_DATA = \
 	map-ISO8859-1 \
 	map-ISO8859-2 \
Index: configure.ac
===================================================================
RCS file: /cvs/xorg/font/util/configure.ac,v
retrieving revision 1.7
diff -u -b -B -r1.7 configure.ac
--- configure.ac	3 Dec 2005 05:49:41 -0000	1.7
+++ configure.ac	8 Dec 2005 19:48:34 -0000
@@ -31,8 +31,12 @@
 AC_PROG_CC
 AC_PROG_INSTALL
 
-mapdir="$libdir/X11/fonts/util"
-AC_SUBST(mapdir)
+DEFAULT_MAPDIR=${libdir}/X11/fonts/util
+AC_ARG_WITH(mapdir,
+	AS_HELP_STRING([--with-mapdir=MAPDIR], [Path to install font maps]),
+	[MAPDIR="$withval"],
+	[MAPDIR="$DEFAULT_MAPDIR"])
+AC_SUBST(MAPDIR)
 
 XORG_MANPAGE_SECTIONS
 XORG_RELEASE_VERSION
